packages/dart/noports_core/pubspec.yaml

```diff
@@ -30,6 +30,7 @@ dependency_overrides:
     git:
       url: https://github.com/atsign-foundation/dartssh2
       ref: trunk
```

Comment:
Do we need to make a transitive dependency explicit like this? My take is that we just need to bump the lock file(s)